src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -1,8 +1,13 @@
 /** @module MapBoxMapProvider */
 
+import MapboxLanguage from '@mapbox/mapbox-gl-language';
+
 import MapProvider from './mapprovider';
 import DOM from '../../../dom/dom';
 
+const SUPPORTED_LANGUAGES =
+      ['en', 'es', 'fr', 'de', 'ar', 'ja', 'ko', 'pt', 'ru', 'zh'];
```

Comment:
It's kind of annoying that mapbox doesn't expose this array to us https://github.com/mapbox/mapbox-gl-language/blob/master/index.js#L168

I guess we'll just have to manually keep this up to date as mapbox adds new languages?

Reply:
Ya, I think that's the simplest